main.py

```diff
@@ -1,19 +1,88 @@
-import pokemon_scanner
+import praw
+import dotenv
+from os import getenv
+from Pokemon_class import Pokemon
+from top_five import get_top_five
+
+dotenv.load_dotenv()
+
+client_secret = getenv("CLIENT_SECRET")
```

Comment:
you'll start seeing this in your linters - but it's good practice in the PEP8 style guide to have all global variables in all upper-case.  so `CLIENT_SECRET` and `USERNAME` and ... etc